### DISCUSSION TITLE
PHP Parse error on "vendor/bin/shlink-installer update"

### DISCUSSION THREAD

**Discussion by morphinestyle** [shlinkio/shlink#1985] [Help wanted] [answered]
### Shlink version

3.7.1 --> 3.7.2

### PHP version

8.2.14

### How do you serve Shlink

Self-hosted Apache

### Database engine

MariaDB

### Database version

10.11

### Summary

Hello!

I can't run the upgrade with "vendor/bin/shlink-installer update", the system return:
`PHP Parse error:  syntax error, unexpected '|', expecting '{' in /home/MYDOMAIN/public_html/current/vendor/shlinkio/shlink-config/functions/functions.php
`
How can i solve?

**Comment by acelaya** [comment, marked as answer]
This happens when running Shlink with an unsupported PHP version.

You probably have multiple versions and the wrong one is being picked when running from the CLI

**Reply by morphinestyle** [reply]
I'm using PHP 8.2.14 since a few weeks.

I had installed Shlink with PHP version 8.2.11 and I had no problems at all, but now with PHP version 8.2.14 I can no longer update to the latest version.

**Reply by morphinestyle** [reply]
This is the full error (line 46):
`PHP Parse error:  syntax error, unexpected '|', expecting '{' in /home/MYDOMAIN/public_html/shlink/vendor/shlinkio/shlink-config/functions/functions.php on line 46`

**Reply by morphinestyle** [reply]
I will try to force PHP

**Reply by morphinestyle** [reply]
Worked!
Thank you Alejandro!
